lib/server.js

```diff
@@ -236,9 +240,23 @@ CoAPServer.prototype.close = function(done) {
     this._lru.reset()
   }
 
+  // cancel cache entry expiry timers
+  for (var k in this._block2Cache) {
+    if (!this._block2Cache.hasOwnProperty(k))
+      continue
```

Comment:
opinion: I would prefer just to throw this whole thing in a block and avoid `continue` since `continue` can generally be avoided